.github/ISSUE_TEMPLATE.md

```diff
@@ -7,10 +7,7 @@ Before opening an issue, make sure there is not a similar issue [already open](h
 ## Expected behavior
 
 ## System Information
-Distribution:
+Distribution and version:
 
-Version:
-
-Metrics-Agent Version:
-
-(paste the output of `do-agent --version`)
+do-agent version:
+(paste the output of `/opt/digitalocean/bin/do-agent --version`)
```

Comment:
This will not work for `0.5.1`. You can find the version info by just running the binary (no `--version`) but not sure how to word that or if there's a better way

Reply:
Yeah I think we may provide a backup for this. This only works on 1.x. If that doesn't work they can/should use the package manager to show the version:

```
# ubuntu/debian
apt-cache policy do-agent

# Fedora, CentOS
yum --cacheonly list do-agent
```

Reply:
Since the output of `--version` is printed either way maybe we can just ask for both the package manager's version _and_ the `--version` output even though the latter might be an error? That's probably the simplest.